lib/rucio/common/utils.py

```diff
@@ -925,10 +923,24 @@ def detect_client_location():
                                          os.environ.get('OSG_SITE_NAME',
                                                         'ROAMING')))
 
+    latitude = os.environ.get('RUCIO_LATITUDE')
+    longitude = os.environ.get('RUCIO_LONGITUDE')
+    if latitude and longitude:
+        try:
+            latitude = float(latitude)
+            longitude = float(longitude)
+        except ValueError:
+            latitude = longitude = 0
+            print('Client set latitude and longitude are not valid.')
+    else:
+        latitude = longitude = 0
+
     return {'ip': ip,
             'ip6': ip6,
             'fqdn': socket.getfqdn(),
-            'site': site}
+            'site': site,
+            'latitude': latitude,
+            'longitude': longitude}
```

Comment:
Not all usages of `detect_client_location` mention these in their documentation (like `rucio.client.replicaclient.ReplicaClient.list_replicas`) leading to false documentation

Reply:
This was motivated a bit by the fact that there is a bit of a movement to use clients rucio-cfg less. Let's have a compromise and have both 😄

Reply:
fixing this now.